build/Targets/VSL.Versions.targets

```diff
@@ -40,32 +48,32 @@
 
   <PropertyGroup>
     <!-- Split the build parts out from the BuildNumber which is given to us by MicroBuild in the format of yyyymmdd.nn
-         where BuildNumberPart1 is yyyymmdd (such as 20160615) and BuildNumberPart2 is nn (which represents the nth build 
-         started that day). So the first build of the day, 20160615.1, will produce something similar to BuildNumberPart1: 20160615,
-         BuildNumberPart2: 01; and the 12th build of the day, 20160615.12, will produce BuildNumberPart1: 20160615, BuildNumberPart2: 12 -->
-    <!-- Additionally, in order to match the MicroBuild v1 behavior, we will only take the last five digits of the BuildNumber, so
-         in the case of 20160615, we will set BuildNumberPart1 to 60615. This will begin failing in the 2017 as the BuildVersion only allows
+         where BuildNumberFiveDigitDateStamp is yyyymmdd (such as 20160615) and BuildNumberBuildOfTheDay is nn (which represents the nth build 
```

Comment:
This implies that `BuildNumberFiveDigitDateStamp` is actually the 8-digit example to the right, not that that is $(BuildNumber.Split('.')[0]), which is I think what it is.